spec/data-model/README.md

```diff
@@ -103,23 +106,25 @@ interface Variant {
 
 interface CatchallKey {
   type: "*";
-  value?: string;
+  value: Pattern;
```

Comment:
This seems like a misunderstanding of this data model entry.

Reply:
Specifically: I think it should be `string` because the intention here is for it to be a different string representing the catchall key in some syntax other than MF2. And it should be optional.